app/js/view/details_view.js

```diff
@@ -35,14 +39,75 @@ export default class DetailsView extends View {
     });
   }
 
+  filterNonAffectedApps(apps, filters) {
+    return apps.filter(app => {
+      return filters.reduce((prev, cur) => {
+        return prev || cur.test(app.manifestURL);
```

Comment:
I'm not sure if ```filter``` is applied to ```manifestURL```, or ```origin```, or some combination thereof, or something else. We should ask Fabrice about this before landing.